### PR TITLE
Fix makeTestMain with classes in the empty package

### PR DESCRIPTION
**Pull request by lolgab**

### CODE REVIEW

File: unit-tests/src/test/scala/RootPackageTest.scala [hunk 1]
```diff
@@ -0,0 +1,3 @@
+object RootPackageTest extends tests.Suite {
```
Comment (sjrd) [inline unit-tests/src/test/scala/RootPackageTest.scala]:
```suggestion
object EmptyPackageTest extends tests.Suite {
```
This whole issue is actually cause by the JVM *empty* package. It's not the root package. If it were the root package, the previous implementation would be correct. Let's use the correct terminology.

**Comment by sjrd** [comment]
Thanks for the contribution. I'm happy to take this in after the above comment has been addressed, even though I expect all of that will become a non-issue once we're through with #1728 and then basing the testing interface on top of `Reflect`, like Scala.js does.

File: unit-tests/src/test/scala/RootPackageTest.scala (hunk 1 above)
Comment (lolgab) [reply unit-tests/src/test/scala/RootPackageTest.scala]: Done ✅

**Comment by lolgab** [comment]
Thank you very much for the review :)
Didn't know there was already work being done in that direction, otherwise I would have not taken this :D

**Pull request merged**